charts/rancher-monitoring/v0.1.4/Chart.yaml

```diff
@@ -7,8 +7,8 @@ maintainers:
 name: rancher-monitoring
```

Comment:
Does the `rancher_min_version` also need to be modified? It's currently at `2.4.6-rc1`.

Reply:
Can we update it to v2.5.0-alpha1?

Reply:
done